repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/CaseTypeEntity.java

```diff
@@ -109,6 +108,11 @@
     @JoinColumn(name = "case_type_id")
     private final List<SearchAliasFieldEntity> searchAliasFields = new ArrayList<>();
 
+    @OneToMany(fetch = LAZY, cascade = ALL, orphanRemoval = true)
+    @Fetch(value = FetchMode.SUBSELECT)
+    @JoinColumn(name = "case_type_id")
+    private final List<RoleToAccessProfileEntity> accessProfiles = new ArrayList<>();
```

Comment:
accessProfiles is not the correct name, it could be a misleading name.  These are not access profiles. These are the mappings between roles to access profiles. 
Can you change the naming here and in other places please?
We could call this `roleToAccessProfilesMappings`

Reply:
Done